schema/vulntology-json-schema-1.0-draft.json

```
@@ -42,22 +42,28 @@
                 }
             ],
             "properties": {
-                "hasEnumeration": {"$ref": "#/definitions/ProductEnumeration"},
+                "hasEnumeration": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/ProductEnumeration"
+                    }
```

@Chris-Turner-NIST Should we have this?
```suggestion
                    },
                    "uniqueItems": true
```